source/sourcing_and_processing_data/processors/what3words.rst

```diff
@@ -0,0 +1,42 @@
+what3words processor
+====================
+
+.. admonition:: This processor is not available by default.
+   :class: important
+
+   Please contact OpenDataSoft support team if you want this processor to be activated in your domain.
+
+This processor uses geographical coordinates to produce a 3 word address.
```

Comment:
Link to the website https://what3words.com/ for reference
And maybe interlinks between the two processors that are related (what3words and reverse what3words)